app/views/events/show.html.erb

```diff
@@ -1,6 +1,6 @@
 <%- model_class = Event -%>
 <div class="page-header">
-  <h1><%= t '.title', :default => model_class.model_name.human.titleize %></h1>
+  <h1><%= @event.kind.humanize %> <%=t '.title', :default => model_class.model_name.human.titleize %></h1>
```

Comment:
I don't think that the event kind should be the first part of the title, but it's good to have it displayed